scss/dialogs/_group-styles.scss

```diff
@@ -1,28 +1,15 @@
-p.group-name,
 input.group-name-input {
-  margin-left: 0px;
-  margin-bottom: 0px;
-  font-size: x-large;
-  width: 100%;
-  border: 0;
-  height: auto;
   background-color: transparent;
-  color: var(--bp4MenuText);
-  user-select: text;
-}
-
-p.group-name {
-  display: flex;
-  p.trucated-name {
-    text-overflow: ellipsis;
-    overflow: hidden;
-    margin-bottom: 0;
-  }
-}
-
-input.group-name-input {
+  border: 0;
   border-bottom: solid;
   border-color: var(--loginInputFocusColor);
+  color: var(--bp4MenuText);
```

Comment:
Can we get rid of BP here? Our plan is to slowly get rid of BluePrint

Reply:
this is variable from us, many duplicate the var/color under a new name in themebase?